libs/wingsdk/src/cloud/bucket.ts

```diff
@@ -49,6 +49,16 @@ export abstract class Bucket extends Resource {
     this.display.title = "Bucket";
     this.display.description = "A cloud object store";
 
+    this._inflightOps.push(
```

Comment:
I *think* (?) that we should wrap this with an API instead of directly have everyone access `_inflightOps`. Something like `this._addInflightMethod()`, but we can change that later...

Reply:
Done